### PR TITLE
add orix conda recipe

### PR DESCRIPTION
**Pull request by dnjohnstone**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dnjohnstone** [comment]
@pc494 please can you confirm you're happy to be a maintainer of this recipe

File: recipes/orix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5ede74e8cded3ad055984f15c2bbb989df6805689ca1b73ac6734b83d4147a7b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/orix/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/orix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - orix.scalar
+    - orix.vector
+  commands:
+    - pytest --pyargs orix
+
+about:
+  home: https://github.com/pyxem/orix
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
```
Comment (marcelotrevisani) [inline recipes/orix/meta.yaml]: It seems that the license is not being packaged with this project
You can package it manually then
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

**Comment by pc494** [comment]
This confirms that I am happy to be listed as a maintainer.

> @pc494 please can you confirm you're happy to be a maintainer of this recipe

File: recipes/orix/meta.yaml (hunk 1 above)
Comment (dnjohnstone) [reply recipes/orix/meta.yaml]: thanks, changed.

File: recipes/orix/meta.yaml (hunk 2 above)
Comment (dnjohnstone) [reply recipes/orix/meta.yaml]: thanks, packaged this.

**Comment by dnjohnstone** [comment]
Thanks for the comments @marcelotrevisani and sorry for making so many slips in this one!

I think this should now be good to go @conda-forge/staged-recipes

**Pull request merged**